packages/amplify-util-mock/src/__e2e_v2__/index-transformer.e2e.test.ts

```diff
@@ -0,0 +1,754 @@
+  await deleteItem('order1', hashKey, '2018-01-01T00:01:01.000Z');
+  await deleteItem('order2', hashKey, '2018-06-01T00:01:01.000Z');
+  await deleteItem('order3', hashKey, '2018-09-01T00:01:01.000Z');
+});
+
+test('Test query with three part secondary key, where sort key is an enum.', async () => {
+  const hashKey = '2018-06-01T00:01:01.000Z';
+  const sortKey = 'UNKNOWN';
+  await createItem('order1', sortKey, 'list1', '2018-01-01T00:01:01.000Z');
+  await createItem('order2', sortKey, 'list2', hashKey);
+  await createItem('order3', sortKey, 'item3', '2018-09-01T00:01:01.000Z');
+  let items = await itemsByCreatedAt(undefined);
+  expect(items.data).toBeNull();
+  expect(items.errors.length).toBeGreaterThan(0);
+  items = await itemsByCreatedAt(hashKey);
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(1);
+  items = await itemsByCreatedAt(hashKey, { beginsWith: sortKey });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(1);
+  items = await itemsByCreatedAt(hashKey, { eq: sortKey });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(1);
+  items = await itemsByCreatedAt(hashKey, { between: [sortKey, sortKey] });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(1);
+  items = await itemsByCreatedAt(hashKey, { gt: sortKey });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(0);
+  items = await itemsByCreatedAt(hashKey, { ge: sortKey });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(1);
+  items = await itemsByCreatedAt(hashKey, { lt: sortKey });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(0);
+  items = await itemsByCreatedAt(hashKey, { le: sortKey });
+  expect(items.data.itemsByCreatedAt.items).toHaveLength(1);
+  items = await itemsByCreatedAt(undefined, { le: sortKey });
+  expect(items.data).toBeNull();
+  expect(items.errors.length).toBeGreaterThan(0);
+  await deleteItem('order1', sortKey, '2018-01-01T00:01:01.000Z');
+  await deleteItem('order2', sortKey, hashKey);
+  await deleteItem('order3', sortKey, '2018-09-01T00:01:01.000Z');
+});
+
+// skipping due to non functioning queryField
+test.skip('Test update mutation validation with three part secondary key.', async () => {
+  const createResponseMissingLastSortKey = await createShippingUpdate({ orderId: '1sttry', itemId: 'item1', name: '42' });
+  expect(createResponseMissingLastSortKey.data.createShippingUpdate).toBeNull();
+  expect(createResponseMissingLastSortKey.errors).toHaveLength(1);
+
+  const createResponseMissingFirstSortKey = await createShippingUpdate({ orderId: '2ndtry', status: 'PENDING', name: '43?' });
+  expect(createResponseMissingFirstSortKey.data.createShippingUpdate).toBeNull();
+  expect(createResponseMissingFirstSortKey.errors).toHaveLength(1);
+
+  await createShippingUpdate({ orderId: 'order1', itemId: 'item1', status: 'PENDING', name: 'name1' });
+  const items = await getShippingUpdates('order1');
+  console.log('items data', items.data);
```

Comment:
There are a few `console.log()`s in the PR. If we don't need them, can you please remove them.